cmd/tsconnect/src/index.ts

```diff
@@ -14,10 +14,22 @@ WebAssembly.instantiateStreaming(
 ).then((result) => {
   // The Go process should never exit, if it does then it's an unhandled panic.
   go.run(result.instance).then(() => handleGoPanic())
+
+  const params = new URLSearchParams(window.location.search)
+  var authKey = ""
```

Comment:
`let` is generally preferred to `var` because it had better scoping rules. https://eslint.org/docs/latest/rules/no-var describes some of the tradeoffs and could be used to automatically check for this, I'm pretty sure we already have it for the admin portal. Will investigate when I'm back.

(I do wish Go, Swift and JS would agree on what `var` and `let` mean, switching between quickly makes my head hurt)

Reply:
Though in this case you can say as a one-liner and have it be `const`:

```
const authKey = params.get("authkey") ?? undefined
```

`??` is the [nullish-coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).